server/services/PolicyService.ts

```diff
@@ -83,17 +83,17 @@ export default class PolicyService {
   /**
    * Calls backend Get Policy API
    */
-  getPolicy = async (req: Request, h: ResponseToolkit): Promise<ServerResponse<GetPolicyResponse>> => {
+  getPolicy = async (req: Request, h: ResponseToolkit): Promise<ServerResponse<DocumentPolicy>> => {
     try {
       const { id } = req.params;
       const params = { policyId: id };
       const { callWithRequest } = await this.esDriver.getCluster(CLUSTER.ISM);
       const getResponse = await callWithRequest(req, "ism.getPolicy", params);
       const policy = _.get(getResponse, "policy", null);
-      const seqNo = _.get(getResponse, "_seq_no", null);
-      const primaryTerm = _.get(getResponse, "_primary_term", null);
+      const seqNo = _.get(getResponse, "_seq_no");
+      const primaryTerm = _.get(getResponse, "_primary_term");
       if (policy) {
-        return { ok: true, response: { id, seqNo, primaryTerm, policy } };
+        return { ok: true, response: { id, seqNo: seqNo as number, primaryTerm: primaryTerm as number, policy: policy as Policy } };
```

Comment:
nit: maybe just +seqNo for convert to number
https://stackoverflow.com/questions/14667713/typescript-converting-a-string-to-a-number